### DISCUSSION TITLE
Save Custom Canvas Sizes

### DISCUSSION THREAD

**Discussion by portablevampire** [Orama-Interactive/Pixelorama#786] [Ideas]
There are a lot of good predefined options for canvas sizes, which is great, but it'd be nice if there was an option to save custom sizes, so that if there's a custom size that's not included in the predefined options that you tend to use a lot, you can easily select it without having to type in the dimensions every time.

I would personally really like this because I use Pixelorama to make 88x31 buttons and 150x20 blinkies (the kind of small graphics that were popular in geocities days) and it would be nice to be able to save those as custom canvas size presets so I can select them easily when I open a new Pixelorama project.

**Comment by Variable-ind** [comment]
> There are a lot of good predefined options for canvas sizes, which is great, but it'd be nice if there was an option to save custom sizes, so that if there's a custom size that's not included in the predefined options that you tend to use a lot, you can easily select it without having to type in the dimensions every time.
> 
> I would personally really like this because I use Pixelorama to make 88x31 buttons and 150x20 blinkies (the kind of small graphics that were popular in geocities days) and it would be nice to be able to save those as custom canvas size presets so I can select them easily when I open a new Pixelorama project.

https://github.com/Orama-Interactive/Pixelorama/pull/819 would add the ability to choose from a recent size
